src/components/language-toggle/language-toggle.ts

```diff
@@ -0,0 +1,5 @@
+module.exports = (language, url) => {
```

Comment:
the name of this file needs to contains "-filter" at the end.
Why?
1/ this package will be rename language-toggle and if you keep this name for this function, import path will become `require("@govuk-one-login/language-toggle/language-toggle");`, which is weird.
2/ It's a good practice to add "-filter" and globally "-typeofyourfunction" at the end of the files when you export a function.

Reply:
updated